Index: t/72pg.t
===================================================================
--- t/72pg.t	(wersja 1843)
+++ t/72pg.t	(kopia robocza)
@@ -14,6 +14,7 @@
 
 plan tests => 8;
 
+DBICTest::Schema->load_classes( 'Casecheck' );
 DBICTest::Schema->compose_connection('PgTest' => $dsn, $user, $pass);
 
 my $dbh = PgTest->schema->storage->dbh;
@@ -74,3 +75,20 @@
 $dbh->do("DROP TABLE testschema.casecheck;");
 $dbh->do("DROP SCHEMA testschema;");
 
+
+BEGIN{
+package DBICTest::Schema::Casecheck;
+
+use base 'DBIx::Class::Core';
+
+use strict;
+use warnings;
+use base 'DBIx::Class';
+
+__PACKAGE__->load_components(qw/PK::Auto Core/);
+__PACKAGE__->table('casecheck');
+__PACKAGE__->add_columns(qw/id name NAME uc_name/);
+__PACKAGE__->set_primary_key('id');
+
+1;
+}
Index: t/lib/sqlite.sql
===================================================================
--- t/lib/sqlite.sql	(wersja 1843)
+++ t/lib/sqlite.sql	(kopia robocza)
@@ -1,6 +1,6 @@
 -- 
 -- Created by SQL::Translator::Producer::SQLite
--- Created on Tue May 23 21:10:54 2006
+-- Created on Fri May 26 15:43:51 2006
 -- 
 BEGIN TRANSACTION;
 
Index: t/lib/DBICTest/Schema.pm
===================================================================
--- t/lib/DBICTest/Schema.pm	(wersja 1843)
+++ t/lib/DBICTest/Schema.pm	(kopia robocza)
@@ -11,7 +11,6 @@
   CD
   Link
   Bookmark
-  #Casecheck
   #dummy
   Track
   Tag
